Sources/iOS/App/CustomViews/UserProfile/UserProfileTopViewController.swift

```diff
@@ -172,6 +173,14 @@ class UserProfileTopViewController: StableTableViewController, Instantiatable, I
             cell.accessibilityIdentifier = "bioCell"
             cells[0].append(cell)
         }
+        if let fields = input.fields {
+            fields.indices.forEach { index in
```

Comment:
中身が一行で済むわけじゃない場合は forEach じゃなくて for-in のほうを使いたいです。
```suggestion
            for index in fields.indices {
```